starter-code/src/main.js

```diff
@@ -1,50 +1,57 @@
-  { name: 'aquaman',         img: 'aquaman.jpg' },
-  { name: 'batman',          img: 'batman.jpg' },
-  { name: 'captain america', img: 'captain-america.jpg' },
-  { name: 'fantastic four',  img: 'fantastic-four.jpg' },
-  { name: 'flash',           img: 'flash.jpg' },
-  { name: 'green arrow',     img: 'green-arrow.jpg' },
-  { name: 'green lantern',   img: 'green-lantern.jpg' },
-  { name: 'ironman',         img: 'ironman.jpg' },
-  { name: 'spiderman',       img: 'spiderman.jpg' },
-  { name: 'superman',        img: 'superman.jpg' },
-  { name: 'the avengers',    img: 'the-avengers.jpg' },
-  { name: 'thor',            img: 'thor.jpg' }
+  { name: "aquaman", img: "aquaman.jpg" },
+  { name: "batman", img: "batman.jpg" },
+  { name: "captain america", img: "captain-america.jpg" },
+  { name: "fantastic four", img: "fantastic-four.jpg" },
+  { name: "flash", img: "flash.jpg" },
+  { name: "green arrow", img: "green-arrow.jpg" },
+  { name: "green lantern", img: "green-lantern.jpg" },
+  { name: "ironman", img: "ironman.jpg" },
+  { name: "spiderman", img: "spiderman.jpg" },
+  { name: "superman", img: "superman.jpg" },
+  { name: "the avengers", img: "the-avengers.jpg" },
+  { name: "thor", img: "thor.jpg" },
+  { name: "aquaman", img: "aquaman.jpg" },
+  { name: "batman", img: "batman.jpg" },
+  { name: "captain america", img: "captain-america.jpg" },
+  { name: "fantastic four", img: "fantastic-four.jpg" },
+  { name: "flash", img: "flash.jpg" },
+  { name: "green arrow", img: "green-arrow.jpg" },
+  { name: "green lantern", img: "green-lantern.jpg" },
+  { name: "ironman", img: "ironman.jpg" },
+  { name: "spiderman", img: "spiderman.jpg" },
+  { name: "superman", img: "superman.jpg" },
+  { name: "the avengers", img: "the-avengers.jpg" },
+  { name: "thor", img: "thor.jpg" }
 ];
 
-$(document).ready(function(){
+$(document).ready(function() {
   var memoryGame = new MemoryGame(cards);
-  var html = '';
-  memoryGame.cards.forEach(function (pic, index) {
+  memoryGame.shuffleCard(cards);
+  var html = "";
+  memoryGame.cards.forEach(function(pic, index) {
     html += '<div class= "card" id="card_' + pic.name + '">';
     html += '<div class="back"';
-    html += '    name="'       + pic.img +  '">';
-    html += '</div>';
+    html += '    name="' + pic.img + '">';
+    html += "</div>";
     html += '<div class="front" ';
     html += 'style="background: url(img/' + pic.img + ') no-repeat">';
-    html += '</div>';
-    html += '</div>';
+    html += "</div>";
+    html += "</div>";
   });
 
   // Add all the div's to the HTML
-  document.getElementById('memory_board').innerHTML = html;
+  document.getElementById("memory_board").innerHTML = html;
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
-});
-});
+  $(".back").on("click", function($e) {
+
+    // $("#card").flip(); //I tried to flip() because I saw the library on the html (doen't work)
```

Comment:
then you don't need all this stuff, you can flip the cards just with this line of code:

```$('#card_fantastic four').children().toggleClass('back front');```